jbpm-wb-dashboard/jbpm-wb-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/AbstractDashboard.java

```diff
@@ -64,15 +68,34 @@
     protected ServerTemplateSelectorMenuBuilder serverTemplateSelectorMenuBuilder;
 
     UberfireBreadcrumbs breadcrumbs;
+    protected Caller<SpecManagementService> specManagementService;
+
+    protected Event<NotificationEvent> notificationEvent;
+
 
     private PerspectiveManager perspectiveManager;
 
     private String detailScreenId;
 
+    @Inject
+    public void setSpecManagementService(final Caller<SpecManagementService> specManagementService) {
+        this.specManagementService = specManagementService;
+    }
+
+    @Inject
+    public void setNotificationEvent(final Event<NotificationEvent> notificationEvent) {
+        this.notificationEvent = notificationEvent;
+    }
+
     public String getPerspectiveId() {
         return perspectiveManager.getCurrentPerspective().getIdentifier();
     }
 
+    protected void displayWarningNotification(String message) {
```

Comment:
Perhaps drop the _warning_ from the method name? _NotificationType_ is also the default, so it could be removed.